### PR TITLE
Mapillary

### PR DESCRIPTION
**Pull request by nboisteault**
Add Mappilary to Lizmap Web Client.

Funded by [Lons-le Saunier](https://www.lonslesaunier.fr/)

### CODE REVIEW

File: library/misc/mapillary/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Mappilary
```
Comment (Gustry) [inline library/misc/mapillary/README.md:1]: Mapillary, not mappillary

Comment (Gustry) [inline library/misc/mapillary/README.md:1]: "Mapillary", please replace in all files, token etc
Many occurrences in this file

File: library/misc/mapillary/src/mapillary.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    minidockopened: e => {
+        if (e.id === 'Mapillary') {
+            // Put OL6 map on top
+            lizMap.mainLizmap.newOlMap = true;
+
+            let init = true;
+
+            // Display Mapillary layers if they exist else init
+            for (const layer of lizMap.mainLizmap.map.getLayers().getArray()) {
+                if (layer.get('title').startsWith('mappilary')) {
```
Comment (Gustry) [inline library/misc/mapillary/src/mapillary.js:43]: Mapillary

**Comment by nboisteault** [comment]
Thanks eagle eye @Gustry !

**Pull request merged**